cpp/src/graphar/general_params.h

```diff
@@ -27,6 +27,7 @@ struct GeneralParams {
   static constexpr const char* kDstIndexCol = "_graphArDstIndex";
   static constexpr const char* kOffsetCol = "_graphArOffset";
   static constexpr const char* kPrimaryCol = "_graphArPrimary";
+  static constexpr const char* kLabelCol = ":LABEL";
```

Comment:
Could we consider renaming the column to start with "_graphAr"?

Reply:
yes, use ":LABEL" because we want to keep the column names consistent with the csv, but of course we can change the name in the arrow table

Reply:
the usage example will be commited in next PR